include/clap/factory/draft/preset-discovery.h

```diff
@@ -193,10 +193,8 @@ typedef struct clap_preset_discovery_location {
    //   eg: file:///home/abique/.u-he/Diva/Presets/Diva (on Linux)
    //   eg: file:///C:/Users/abique/Documents/u-he/Diva.data/presets/ (on Windows)
    //
-   // - plugin://<resource-id> for presets which are bundled inside the plugin itself.
-   //   The resource-id string is defined by the plugin and should be unique to avoid clashing
-   //   with other providers, because when preset_load.from_uri() will be called, the
-   //   provider's origin won't be known.
+   // - plugin:// for presets which are bundled within the plugin DSO.
+   //   The path component may be empty.
```

Comment:
It may seem obvious but just to prevent future hosts from implementing it incorrectly it wrong it might be worth explicitly mentioning that the `plugin://[path]` URIs are local to this specific .clap plugin, and that multiple plugins may use the same `plugin://` URI.

Reply:
Do you have a sentence to suggest?
I could also say that the path component must be empty.

Reply:
I'll force it to be exactly `plugin://` that simplifies everything.

Reply:
One empty path per plugin should also work. The only use cases I can think of where exposing multiple internal preset locations would be useful are for grouping built in presets into multiple sound packs, and for providing internal preset data for plugins using other plugin APIs (but that seems like both a fringe use case, and something that wouldn't consistently work in the first place since you may lack up to date information on that plugin's internal presets).

Reply:
Maybe we just go with the simpler approach and if we need more, then we improve it.